src/pages/components/checkbox/usage.mdx

```diff
@@ -37,35 +52,191 @@ The default view of a set of checkboxes is having no option selected.
 </Column>
 </Row>
 
-<Caption>By default, checkboxes are unchecked.</Caption>
+<Caption>Example of a full page filter list.</Caption>
+
+### When not to use
+
+If a user can select only one option from a list, radio buttons should be used instead of checkboxes. Checkboxes allow the user to select multiple items in a set whereas radio buttons allow the user to select only one option.
+
+<DoDontRow>
+  <DoDont caption="Do use radio buttons when only one item can be selected.">
+
+![Do use radio buttons when only one item can be selected.](images/checkbox-usage-2-do.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Don't use checkboxes when only one item can be selected.">
+
+![Don't use checkboxes when only one item can be selected.](images/checkbox-usage-2-dont.png)
+
+  </DoDont>
+</DoDontRow>
+
+## Formatting
+
+### Anatomy
+
+The checkbox component is comprised of a label and a checkbox. If there is a group of checkboxes a heading can be added.
+
+<Row>
+<Column colLg={8}>
+
+![The default state for checkboxes is unchecked.](images/checkbox-usage-3.png)
+
+</Column>
+</Row>
+
+1. **Heading (optional):** Communicates what needs to be selected below.
```

Comment:
I think we might need to flag this as required potentially, but I think it's right to say that this doesn't have to present itself visually.

In code this group might look like:

```html
<h3 id="group-label">Languages</h3>
<div role="group" aria-labelledby="group-label">
  <ul>
    <li>
      <input id="option-1" type="checkbox" />
      <label for="option-1">Node.js</label>
    </li>
    <li>
      <input id="option-2" type="checkbox" />
      <label for="option-2">Go</label>
    </li>
    <li>
      <input id="option-3" type="checkbox" />
      <label for="option-3">Java</label>
    </li>
  </ul>
</div>
```

And as a result we do need a label for the group, at the very least in code but ideally presented to both user groups.

Reply:
In code, if there was one checkbox present would you need a heading for it in code? (ex: terms and conditions). if not, then maybe we just need to clarify there that headings for groups of checkboxes is required.

Reply:
@laurenmrice good question, I don't know if we would say heading necessarily but we would need a label. If you look at that markup, we would say that the label for the checkbox is the `<label>` element, for example:

```html
<input id="option-3" type="checkbox" />
<label for="option-3">Java</label>
```

We would say that the label for the checkbox is `Java`.

The group also has a label, but instead, it uses `aria-labelledby` instead of the `<label>` element. The `aria-labelledby` attribute points to the heading `<h3>` tag and treats its text as the label for the group.

Reply:
Okay, For this we can say something about how visually its not required unless there is a grouping of two or more checkboxes and also say that in code it needs to be required even if its just one checkbox and can reference the code tab for my information.

Reply:
Is it okay that we are labeling the anatomy of the checkbox as 1. Heading. 2. Checkbox and 3. Label ? If in fact `<label>` is referencing the "heading". In the react demo and on the component switcher demo the text is showing how I am currently labeling them here.

Reply:
@laurenmrice I think it breaks down into:

- A checkbox input requires a label
- A group of checkbox inputs requires a grouping label to describe the group of checkbox inputs
- A label is always required for a checkbox input and group of checkboxes, but in rare cases it does not need to be visible. To understand how to label something without it showing up visually, see ___.


![image](https://user-images.githubusercontent.com/3901764/80030037-4b8d9500-84ad-11ea-9cf0-b8ff5f98e941.png)

Reply:
In the newest build, I changed all the content on the page to reflect this. ^